scala/Dummy.java

```diff
@@ -0,0 +1,5 @@
+public class Dummy {
```

Comment:
I'd like to put this into some namespace we control on the off chance it causes strange errors for users who make a Dummy class in an example.

Reply:
I also thought about this from a different perspective of WTF in 6 months. Maybe something like `PlaceHolderClassToCreateEmptyJarForScalaImport` with a comment above pointing to the bazel issue

Reply:
I renamed it to `PlaceHolderClassToCreateEmptyJarForScalaImport`, thanks for the suggestions!